option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message PetRaidFightRequest
{
 required uint32 raid_id = 1;
 repeated uint32 pos = 2;
 repeated U64Type guid = 3;
}
